src/fwrite.c

```diff
@@ -906,19 +914,19 @@ void fwriteMain(fwriteMainArgs args)
       // compress buffer if gzip
       if (args.is_gzip && !failed) {
         myzbuffUsed = zbuffSize;
-        if (verbose) {DTPRINT(_("z_stream for data (%d): "), 2); print_z_stream(&mystream);}
-        int ret = compressbuff(&mystream, myzBuff, &myzbuffUsed, myBuff, (size_t)(ch-myBuff));
-        if (verbose) {DTPRINT(_("z_stream for data (%d): "), 3); print_z_stream(&mystream);}
+        if (verbose) {DTPRINT(_("z_stream for data (%d): "), 3); print_z_stream(mystream);}
```

Comment:
question: supposing this works, would you then aim to remove all the extra verbose printing for tracing in the next release? (less maintenance overhead / tidier code vs. ease of setting up the in-depth tracing should a similar error recur later on)

curious your thinking

Reply:
yes will just remove the extra tracing.  can always get it back from the history should we need it again